CMakeLists.txt

```diff
@@ -685,15 +685,21 @@ else()
   include_directories(Externals/mbedtls/include)
 endif()
 
-find_package(CURL)
-if(CURL_FOUND)
-  message(STATUS "Using shared libcurl")
-  include_directories(${CURL_INCLUDE_DIRS})
+# The libcurl we currently use has a harcoded certificate location
+# It can only be used on distributions where SSL certificates actually are at that path.
+if (NOT UNIX OR EXISTS "/etc/ssl/certs/ca-certificates.crt")
+  find_package(CURL)
+  if(CURL_FOUND)
+    message(STATUS "Using shared libcurl")
+    include_directories(${CURL_INCLUDE_DIRS})
+  else()
+    message(STATUS "Using static libcurl from Externals")
+    add_subdirectory(Externals/curl)
+    set(CURL_LIBRARIES curl)
+    include_directories(BEFORE Externals/curl/include)
+  endif()
 else()
-  message(STATUS "Using static libcurl from Externals")
-  add_subdirectory(Externals/curl)
-  set(CURL_LIBRARIES curl)
-  include_directories(BEFORE Externals/curl/include)
+  find_package(CURL REQUIRED)
```

Comment:
You might want to put `message(STATUS "Using shared libcurl")` here. Also, why is `include_directories` used above but not here?

Reply:
I *somehow* missed that.